sdk/eventhub/Azure.Messaging.EventHubs.sln

```diff
@@ -35,14 +35,6 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{6221D9A4-8563-4B8B-86B2-73585D54F840}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I do not know what caused this, but things seem happy...

Reply:
:magic_wand: